docs/openapi/components/schemas/common/AgricultureParcelDelivery.yml

```diff
@@ -89,44 +89,53 @@ properties:
       $ref: ./AgriculturePackage.yml
     $linkedData:
       term: agriculturePackage
-      '@id': https://w3id.org/traceability#AgriculturePackage
+      '@id': https://w3id.org/traceability#itemShipped
```

Comment:
does not resolve to a term definition: https://w3id.org/traceability#itemShipped